### ISSUE TITLE
Error in readRDS(parameter_filename) : error reading from connection

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/Cer2016#29]
This simple code fails:

```
parameter_filename <- find_path("article_0_0_0_0_0.RDa")
assert(file.exists(parameter_filename))
readRDS(parameter_filename)
```

Error:

```
Error in readRDS(parameter_filename) : error reading from connection
```

When doing this:

```
parameter_filename <- find_path("toy_example_1.RDa")
file.exists(parameter_filename)
readRDS(parameter_filename)
```

Hypothesis: cannot open files from the cluster :-(

**Comment by richelbilderbeek** [comment]
I got a file opened, `article_0_1_4_0_2.RDa`.

It just appears so that some files are corrupt.

**Comment by richelbilderbeek** [comment]
I found the problem: in the old parameter file version (`0.1`), I still used long parameter identifiers:

![29](https://cloud.githubusercontent.com/assets/2098230/15190755/e0f9c28e-17b1-11e6-952a-e0f0b2e6a86c.png)

Time to add some backwards compatibility....

**Comment by richelbilderbeek** [comment]
Darn, I have not update the the parameter file versions! I should have used Travis and @lintr-bot from the direct start of the project....

**Comment by richelbilderbeek** [comment]
I found some versions *being* corrupted, and -AFAIKS- react correctly on this.

**Comment by richelbilderbeek** [comment]
I just noted something weird:

![weird](https://cloud.githubusercontent.com/assets/2098230/15212974/584bcc46-1844-11e6-911c-f40b3dd43e43.png)

**Comment by richelbilderbeek** [comment]
Note that the table is not tidy: the column and row names are not shown in bold. This is fixed in commit 07f8730d62f88c601e54d1344c9030d8835ba9dc

**Comment by richelbilderbeek** [comment]
Nah, it works just fine.
